config.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 	TunaGeoDBPath          string
 	TunaMeasureBandwidth   bool
 	TunaMeasureStoragePath string
+	TunaReverseMinBalance  string
```

Comment:
We can use `TunaMinBalance` because everything here is reverse

Reply:
Yes, modified